sagemaker_processing/fairness_and_explainability/text_explainability_sagemaker_algorithm/byo_blazingtext_model_hosting.ipynb

```diff
@@ -0,0 +1,892 @@
+{
```

Comment:
<p>Formatting nit: is it intentional to have backticks in the file structure in front of "-- blazing_text" and "-- wsgi.py"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Yes its intentional.